policy/modules/system/systemd-homed.fc

```diff
@@ -0,0 +1,27 @@
+#
+# homed file context
+#
+
+/usr/lib/systemd/systemd-homed                            --  gen_context(system_u:object_r:systemd_homed_exec_t,s0)
+/usr/lib/systemd/systemd-homework                         --  gen_context(system_u:object_r:systemd_homework_exec_t,s0)
+/usr/lib/systemd/system/systemd-homed-activate\.service   --  gen_context(system_u:object_r:systemd_homed_unit_file_t,s0)
+/usr/lib/systemd/system/systemd-homed\.service            --  gen_context(system_u:object_r:systemd_homed_unit_file_t,s0)
+
+/var/home/(.+)\.home                                      --  gen_context(system_u:object_r:systemd_homed_crypto_luks_t,s0)
+/var/home/(.+)\.homedir                                   --  gen_context(system_u:object_r:user_home_dir_t,s0)
+
+/var/lib/systemd/home/(.+)\.identity                      --  gen_context(system_u:object_r:systemd_homed_record_t,s0)
+/var/lib/systemd/home/local\.private                      --  gen_context(system_u:object_r:systemd_homed_record_t,s0)
+/var/lib/systemd/home/(.+)\.public                        --  gen_context(system_u:object_r:systemd_homed_record_t,s0)
+/var/lib/systemd/home/local\.public                       --  gen_context(system_u:object_r:systemd_homed_record_t,s0)
+/var/lib/systemd/home                                     -d  gen_context(system_u:object_r:systemd_homed_library_dir_t,s0)
+
+/var/run/cryptsetup                                       -d  gen_context(system_u:object_r:systemd_homed_cryptsetup_dir_t,s0)
```

Comment:
only /run is used since commit 1f76e522ab
the type name does not seem to be indicative